src/components/issues/issue-list-search-input.tsx

```diff
@@ -1,4 +1,4 @@
-"use client";
+import "client-only";
```

Comment:
It doesn't, it just enforces that the component is only from another client component. 

This is because it accepts some props that are required but can't be passed from a server component (the `onSearch` prop).

Reply:
Also, this component only make sense to be used if called from within a client component.